.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-json
+      - id: check-added-large-files
+      - id: check-ast
+      - id: mixed-line-ending
+      - id: requirements-txt-fixer
+        files: requirements.*\.txt
```

Comment:
Any reason not to include all of the checks in the [corresponding changes](https://github.com/ThePalaceProject/circulation-core/pull/47/commits/d66257389c0f06da9e3141e7d20277ec45d3ee85#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9) to `core`? And vice versa?

Reply:
Nope! Good catch. I thought i had. They should be synced up now.